### ISSUE TITLE
RFE: Allow building with external dSFMT library

### ISSUE THREAD

**Issue by opoplawski** [gnudatalanguage/gdl#798]
Fedora packaging requires building with system versions of libraries if possible.  Please and a cmake flag to allow building with an external dSFMT library.

**Comment by GillesDuvert** [comment]
Do you have such a library?
Which would have the 'vector' issue?

**Comment by opoplawski** [comment]
Yes, Fedora ships dSFMT.  It does look like we would need to patch that as well, but that's an issue for Fedora (me) to deal with.

**Comment by GillesDuvert** [comment]
In this case, aside the patch you mention, there could be other problems for GDL, I vaguely remember the compilation with the pristine dSFMT code was not possible.
